docs/x509.rst

```diff
@@ -181,6 +181,71 @@ X.509 Certificate Object
 
         For version 3 X.509 certificates.
 
+.. class:: Attribute
+
+    .. versionadded:: 0.8
+
+    .. attribute:: oid
+
+        :type: :class:`ObjectIdentifier`
+
+        The attribute OID.
+
+    .. attribute:: value
+
+        :type: :class:`str`
```

Comment:
Err, oops, this is not the dotted representation, but it is a string (not bytes) because it can return unicode (that has been decoded from raw code points).